pkg/validator/output.go

```diff
@@ -25,6 +28,17 @@ const (
 	PolarisOutputVersion = "1.0"
 )
 
+var (
+	successMessage = "🎉 Success"
+	dangerMessage  = "❌ Danger"
+	warningMessage = "😬 Warning"
```

Comment:
```suggestion
	warningMessage = "⚠️ Warning"
```
Maybe? 🤔

Reply:
That's what I tried first!

Strangely the ⚠️ emoji gets mangled in my terminal. I wonder if I've just got a bad font, or if it's something that would affect other people...